scrutiny.json

```diff
@@ -28,55 +29,35 @@
                 ],
                 "ports" : {
                     "8080": {
-                        "description": "Port for scrutiny's web interface and API.",
+                        "description": "Port for Scrutiny's web interface and API.",
                         "host_default": 8080,
                         "label": "WebUI HTTP port",
                         "ui": true
                     }
                 },
                 "volumes": {
                     "/config": {
```

Comment:
if this is supposed to be the path within the container, this is incorrect - it should be `/opt/scrutiny/config`

Reply:
@AnalogJ Thanks, yes I did just leaving that as-was and failed to check. I wondered why the syslog kept apparently indicating the creation of a new scrutiny config. Sorted via a follow-up commit.
Cheers.